src/cogs/logging_name_changes.py

```diff
@@ -21,35 +26,27 @@ async def on_member_update(self, before, after):
             username_after = after.nick
 
         if before.nick != after.nick and before.nick is not None:
-            embed = discord.Embed(
-                title="<:grey_exclamation:1044305627201142880> Name Change",
-                description=f"Changed by: {before}.",
-                color=discord.Color.dark_grey(),
-                timestamp=datetime.utcnow(),
-            )
-            embed.set_thumbnail(url=after.avatar)
-            embed.add_field(name="Before", value=username_before, inline=True)
-            embed.add_field(name="After", value=username_after, inline=True)
-
-            logs_channel = await self.bot.fetch_channel(
-                log_channel["user_log"]
-            )  # ADMIN user log
-            await logs_channel.send(f"{username_after.mention}", embed=embed)
+            embed = embed_name_change(before, after, username_before, username_after)
```

Comment:
wonder if the embed functions can become a single embed function that you parametize?